mergify_engine/web/root.py

```diff
@@ -66,6 +67,7 @@ async def shutdown() -> None:
 async def redis_errors(
     request: requests.Request, exc: aredis.exceptions.ConnectionError
 ) -> responses.JSONResponse:
+    statsd.increment("engine.redis.connection-errors")
```

Comment:
```suggestion
    statsd.increment("redis.client.connection.errors")
```
Then we can use this in every service and apply tags.